dune-project

```diff
@@ -34,13 +30,15 @@ new subcommands for capturing, checking, and approving the output of tests.")
 
 (package
   (name testo-lwt)
-  (synopsis "Extension of the Alcotest library for Lwt")
+  (synopsis "Test framework for OCaml, Lwt variant")
   (description "\
 Use this if the tests return Lwt promises and you can't make them synchronous
 by wrapping them in 'Lwt_main.run'.")
   (depends
     (ocaml (>= 4.08.0))
+    alcotest
```

Comment:
maybe explain here why we still depend on alcotest?

Reply:
ok